int-tests/src/test-integration/java/io/neow3j/protocol/Neow3jReadOnlyIntegrationTest.java

```diff
@@ -621,6 +624,38 @@ public void testGetStorage() throws IOException {
         assertThat(storage, is("QAFBAighAzpNBRsEt/wCMNKxqu39WoS+J5pTYac1jbZlrXhXeH8bIQA="));
     }
 
+    @Test
+    public void testFindStorage() throws IOException {
+        NeoFindStorage.FoundStorage foundStorageActual = getNeow3j()
+                .findStorage(GAS_HASH, "", BigInteger.ZERO)
+                .send()
+                .getFoundStorage();
+
+        assertFalse(foundStorageActual.isTruncated());
+        assertThat(foundStorageActual.getNext(), is(BigInteger.valueOf(4)));
```

Comment:
@mialbu I assume you found out by experimenting that the found storage entries are truncated and that the next index is `4`. 
Do you know if the number of the entries returned in one call of this RPC method is configuratble in the node settings, e.g., like the iterator settings?

Reply:
Actually, if a response is not truncated, the value `next` will directly provide you the number of existing storage entries  for the provided prefix. So, I guess that value is useful even if a response is truncated.

![jiffy](https://github.com/neow3j/neow3j/assets/53603111/d4b74474-43f3-440e-a5c3-225ff1d30b19)

Reply:
oh i see. many thanks for the explanation 🙏